### DISCUSSION TITLE
Wie eine Sprache entfernen?

### DISCUSSION THREAD

**Discussion by Hanibal1963** [dotnet/ResXResourceManager#654] [Q&A] [answered]
Hallo Gemeinde!
Ich habe versehentlich eine Sprache hinzugefügt die ich nicht möchte und nun weis ich nicht wie ich diese wieder entfernen kann.

![ResX ressource Manager Menu](https://github.com/user-attachments/assets/fd1c8e58-9eb6-421d-9409-c66f520840c8)

Im Ressource Manager kann man die sprache ausblenden indem der Haken entfernt wird (siehe oben) aber Visual Studio bemängelt die fehlende Übersetzung (siehe unten)

![ResX ressource Manager Fehler](https://github.com/user-attachments/assets/477c0b00-8718-4875-9837-603003b9a038)

Meine Frage ist nun wie ich das umgehen kann.
Kann mir da jemand helfen?

Danke im Voraus

**Comment by tom-englert** [comment, marked as answer]
Einfach die *.en.resx Datei(en) löschen

**Reply by Hanibal1963** [reply]
Das habe ich bereits getan.
Im Ressource Manager ist die Sprache zwar nicht ausgewählt aber trotzdem vorhanden bzw. aktiv.
In Visual Studio wird die Warnung trotzdem weiter angezeigt.

**Reply by tom-englert** [reply]
Auch die Solution danach neu geladen?

**Reply by Hanibal1963** [reply]
ja und zwischenzeitlich auch schon neu gestartet
